x-pack/test/case_api_integration/security_and_spaces/tests/common/configure/get_configure.ts

```diff
@@ -106,5 +139,145 @@ export default ({ getService }: FtrProviderContext): void => {
         })
       );
     });
+
+    describe('rbac', () => {
+      it('should return the correct configuration', async () => {
+        await createConfiguration(supertestWithoutAuth, getConfigurationRequest(), 200, {
+          user: secOnly,
+          space: 'space1',
+        });
+
+        await createConfiguration(
+          supertestWithoutAuth,
+          { ...getConfigurationRequest(), owner: 'observabilityFixture' },
+          200,
+          {
+            user: obsOnly,
+            space: 'space1',
+          }
+        );
+
+        for (const scenario of [
+          {
+            user: globalRead,
+            numberOfExpectedCases: 2,
+            owners: ['securitySolutionFixture', 'observabilityFixture'],
+          },
+          {
+            user: superUser,
+            numberOfExpectedCases: 2,
+            owners: ['securitySolutionFixture', 'observabilityFixture'],
+          },
+          { user: secOnlyRead, numberOfExpectedCases: 1, owners: ['securitySolutionFixture'] },
+          { user: obsOnlyRead, numberOfExpectedCases: 1, owners: ['observabilityFixture'] },
+          {
+            user: obsSecRead,
+            numberOfExpectedCases: 2,
+            owners: ['securitySolutionFixture', 'observabilityFixture'],
+          },
+        ]) {
+          const configuration = await getConfiguration({
+            supertest: supertestWithoutAuth,
+            query: { owner: scenario.owners },
+            expectedHttpCode: 200,
+            auth: {
+              user: scenario.user,
+              space: 'space1',
+            },
+          });
+
+          ensureSavedObjectIsAuthorized(
+            configuration,
+            scenario.numberOfExpectedCases,
+            scenario.owners
+          );
+        }
+      });
+
+      for (const scenario of [
+        { user: noKibanaPrivileges, space: 'space1' },
+        { user: secOnly, space: 'space2' },
+      ]) {
+        it(`User ${scenario.user.username} with role(s) ${scenario.user.roles.join()} and space ${
+          scenario.space
+        } - should NOT read a case`, async () => {
+          // super user creates a configuration at the appropriate space
+          await createConfiguration(supertestWithoutAuth, getConfigurationRequest(), 200, {
+            user: superUser,
+            space: scenario.space,
+          });
+
+          // user should not be able to read configurations at the appropriate space
+          await getConfiguration({
+            supertest: supertestWithoutAuth,
+            expectedHttpCode: 403,
+            auth: {
+              user: scenario.user,
+              space: scenario.space,
+            },
+          });
+        });
+      }
+
+      it('should respect the owner filter when having permissions', async () => {
+        await Promise.all([
+          await createConfiguration(supertestWithoutAuth, getConfigurationRequest(), 200, {
```

Comment:
I think we can remove the `await` next to the calls within the `Promise.all` right?